draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -368,10 +386,12 @@
             <t>
                 If the offerer inserts the SDP 'setup' attribute with an 'actpass' or 'passive' attribute value, the
                 offerer MUST be prepared to receive a DTLS ClientHello message (if a new DTLS association
-                is established by the answerer) from the answerer before the offerer receives the SDP answer.
+                is established by the answerer) from the answerer before the offerer receives the SDP answer. In such
+                case the offerer MAY cache the ClientHello message, but the offerer MUST NOT initiate a DTLS

```

Comment:
Does anyone object to the text? I do realise it's not what everyone would work, but unless someone comes up with something else I think it's a good compromise that will allow us to move forward (it obviously would have to be agreed on the list too).